import/popups/PopupOpener.qml

```diff
@@ -42,6 +42,8 @@ Timer {
         anchor: Notice.Center
     }
 
+    property var alertData
```

Comment:
Worth creating new tasks to fix confirm and prompt as well as those should be similarly broken. Should checked at least.

Reply:
Will check